test/library/standard/Types/copyable-sync-single-string.chpl

```diff
@@ -0,0 +1,7 @@
+var x: sync string;
+param p = isCopyableType(x.type);
+compilerWarning("isCopyableType(sync string) = " + p:string);
+var y: sync string;
+param p2 = isCopyableType(y.type);
+compilerWarning("isCopyableType(single string) = " + p2:string);
```

Comment:
I don't think these need to be `compilerWarning`s, they could just be writelns now